inference-engine/tests_deprecated/behavior/vpu/shared_tests_instances/plugin_tests/vpu_test_data.hpp

```diff
@@ -112,6 +112,8 @@ const std::vector<BehTestParams> deviceAgnosticConfigurations = {
         {MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES, "MYRIAD"},
         {VPU_CONFIG_KEY(HW_STAGES_OPTIMIZATION), CONFIG_VALUE(YES)}
     }),
+
+    BEH_MYRIAD.withConfig({{VPU_MYRIAD_CONFIG_KEY(MOVIDIUS_DDR_TYPE), VPU_MYRIAD_CONFIG_VALUE(NOT_PROG)}}),
```

Comment:
Is it the only valid option?

Reply:
No, but I'm worried about the devices. If we use the wrong type of memory, the device in the test environment will be broken.

Reply:
Could you please add a comment about that. If someone will refactor these tests it's more likely he'll try to add new options there.

The better solution is use mock device for those tests, if possible, but it'll take more time I guess.

Reply:
As far as I understand, we have unit tests like this. All types of options are presented there.

Reply:
> it'll take more time I guess.

and this is deprecated infrastructure